packages/client/src/components/interface/Header/Header.tsx

```diff
@@ -343,6 +341,7 @@ class HeaderComp extends React.Component<IProps, IState> {
     )
   }
 
+  // OLD menu items in header?? Can be removed?
```

Comment:
@jpye-finch this function is required as it decides what the landing page should be in Team view for different user types.  Can you remove this comment please?